src/prep.ts

```diff
@@ -0,0 +1,25 @@
+import { spawn } from 'child_process'
+import * as fs from 'fs'
+import * as detectIndent from 'detect-indent'
+
+import * as fission from './index'
+import * as util from './util'
+
+export function getEnvVars () : Map <string, string> {
+  let map = new Map()
+  map.set('COMMIT', util.getCommit())
+  map.set('BRANCH', util.getBranch())
+  map.set('PLATFORM', util.getPlatform())
+  return map
+}
+
+export function writeElectronBuilderEnvFile (path : string, envVars : Map <string, string>) {
+  envVars.forEach((val, key) => {
+    // console.log('key', key, 'val', val)
+    fs.appendFileSync(path, `${key}=${val}\n`)
```

Comment:
Probably better to construct whole string with loop, then append once to file, as opposed to looping through vars and appending for each.

Seems more error prone to write multiple times, no?

Reply:
Possible. Made the optimization